js/main.js

```diff
@@ -78,12 +84,49 @@ var renderMapPin = function (template, ad) {
 
   mapElement.style.left = ad.location.x - (mapElementImg.width / 2) + 'px';
   mapElement.style.top = ad.location.y - mapElementImg.height + 'px';
+
   return mapElement;
 };
 
+var renderPopup = function (template, ad) {
+  var popupElement = template.cloneNode(true);
+  var popupElementImg = popupElement.querySelector('.popup__avatar');
+  var popupElementTitle = popupElement.querySelector('.popup__title');
+  var popupElementAddress = popupElement.querySelector('.popup__text--address');
+  var popupElementPrice = popupElement.querySelector('.popup__text--price');
+  var popupElementType = popupElement.querySelector('.popup__type');
+  var popupElementCapacity = popupElement.querySelector('.popup__text--capacity');
+  var popupElementTime = popupElement.querySelector('.popup__text--time');
+  var popupElementFeatures = popupElement.querySelector('.popup__features').children;
+  var popupElementDescription = popupElement.querySelector('.popup__description');
+  var popupElementPhotos = popupElement.querySelector('.popup__photos');
+  var popupElementPhoto = popupElement.querySelector('.popup__photo');
+
+  popupElementImg.src = ad.author.avatar;
+  popupElementTitle.textContent = ad.offer.title;
```

Comment:
Когда стоит заводить переменную для DOM элемента: если код используется больше двух-трех раз, то стоит его заключить в переменную, в противном случае можно вызывать как есть.
Таким правилом я пользуюсь. У тебя каждая из них используется единажды, потому кажется избыточным дополнительно создавать переменные.

Можем продолжить так, ничего дурного в этом нет: 
```suggestion
  popupElement.querySelector('.popup__avatar').src = ad.author.avatar;
  popupElement.querySelector('.popup__title').textContent = ad.offer.title;
```

Reply:
Заметил, что для именования узлов ты используешь как постфикс `node` так и `element`. Стоит выбрать что-то одно, иначе можем не пройти по критерию единообразия